node-runner-cli/Pipfile

```diff
@@ -6,7 +6,7 @@ name = "pypi"
 [packages]
 core_client = {git = "https://github.com/radixdlt/python-core-client", editable = true, ref = "v1.1.1"}
 requests = "==2.26.0"
-PyYAML = "==5.4.1"
+PyYAML = "~=6.0.0"
```

Comment:
Out of interest, why do we need the change here?
And we likely want to fix the version number; to mitigate targeted supply chain attacks.

Reply:
So there is this bug that I spent almost a day with:

[Disabling PEP 517 processing is invalid: project specifies a build backend of setuptools.build_meta:__legacy__ in pyproject.toml](https://github.com/yaml/pyyaml/issues/596)

Other tools have this issue ([1](https://stackoverflow.com/questions/70537534/pipenv-exceptions-resolutionfailure-error-disabling-pep-517-processing-is-inva), [2](https://stackoverflow.com/questions/65658570/pipenv-install-fails-on-cryptography-package-disabling-pep-517-processing-is-i)). It seems to be a problem with pipenv: [https://github.com/pypa/pipenv/issues/4481](https://github.com/pypa/pipenv/issues/4481)

I tried removing the `editable = true` part but I got the same error. Updating to PyYAML 6 seems to fix this issue.

Reply:
Ahh okay, thanks. Can we fix the version though, to prevent an auto-acceptance of unreviewed dependency updates? 👍

Reply:
sure